di_website/templates/includes/cards/card-job.html

```diff
@@ -6,8 +6,8 @@
         <h3 class="card__title"><span>{{ vacancy.title }}</span></h3>
         <span class="card__meta">{{ vacancy.location }}</span>
         <ul class="card__list">
-            <li><span>Salary scale: </span>{{ vacancy.salary_scale }}</li>
-            <li><span>Closing date: </span>{{ vacancy.application_close|date:"j F Y" }}</li>
+            <li><span></span>{{ vacancy.salary_scale }}</li>
+            <li><span></span>{{ vacancy.application_close|date:"j F Y" }}</li>
```

Comment:
Nope ... why have you removed the labels? hmm... are you aware that you can use conditional operators in Django templates??

Reply:
Ohh yes I see it now, removing the labels doesn't really solve the issue. Yes to the conditional operators, let me resolve this and push the changes, thank you.